openedx/features/edly/models.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+from django.db import models
+from model_utils.models import TimeStampedModel
+from organizations.models import Organization
+
+ALPHANUMERIC_VALIDATOR = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
+
+
+class EdlyOrganization(TimeStampedModel):
+    """
+    EdlyOrganization model for Edly.
+    """
+    name = models.CharField(max_length=255)
+    slug = models.CharField(max_length=100, unique=True, validators=[ALPHANUMERIC_VALIDATOR])
```

Comment:
`name` and `slug` fields should have similar length https://github.com/edx/edx-organizations/blob/master/organizations/models.py

Reply:
@asadmanzoor93 as discussed https://github.com/edly-io/edx-platform/pull/71#issuecomment-623958450 we will keep this length upto 50 characters (since this slug will be used in the URL's).

Reply:
Done